R/evalParsRollapply.R

```diff
@@ -98,23 +241,5 @@ evalParsTS <- function(par.matrix, object,
   results
 }
 
-## Calculate objective function on a rolling window
-evalParsRollapply <- function(par.matrix, object,
-                              width = 30,
-                              objective = hydromad.getOption("objective"),
-                              parallel = hydromad.getOption("parallel")[["evalParsTS"]],
-                              filehash.name = tempfile()) {
-  fun <- function(thisMod, width, objective) {
-    rollapply(cbind(Q = observed(thisMod), X = fitted(thisMod)),
-      width = width, by.column = FALSE,
-      FUN = objFunVal, objective = objective
-    )
-  }
-
-  evalParsTS(par.matrix, object, fun,
-    length.out = length(observed(object)) - width + 1,
-    parallel = parallel,
-    filehash.name = filehash.name,
-    width = width, objective = objective
-  )
-}
+#' @import utils
+utils::globalVariables(c("cl", "p"))
```

Comment:
As for evalPars - ideally p would not be a global variable

Reply:
Similar to the above, R CMD check complained about this so I added it as a global variable to satisfy the checks:

`evalPars: no visible binding for global variable ‘p’`

I followed suggestions by the R community: [https://community.rstudio.com/t/how-to-solve-no-visible-binding-for-global-variable-note/28887/3](https://community.rstudio.com/t/how-to-solve-no-visible-binding-for-global-variable-note/28887/3)